src/more-infos/more-info-configurator.html

```diff
@@ -120,7 +144,7 @@
 
       fieldInput: {
         type: Object,
-        value: function () { return {}; },
+        computed: 'setDefaultFieldInputs(stateObj)',
```

Comment:
This is not good, it means that if an error comes in, we wipe all the fields. computed functions are called with `(newVal, oldVal)`. We should only generate defaults if the new state object has a different entity id than the old one or there was no old one.

Actually, would have been nice if this was split out in a checkbox PR and a default values PR… Where do these default values come from?